README.md

```diff
@@ -179,40 +187,73 @@ in `onegov.yml`:
     GRANT ALL PRIVILEGES ON DATABASE onegov TO dev;
     ALTER DATABASE onegov SET timezone TO 'UTC';
 
+Use the 'exit' command once your done to quit the postgres cli.
+
+    postgres=# exit
+
+
 Onegov cloud uses one database for all applications and instances.
 
-## Setting up the application(s)
 
-**Org, Town, Town6, Agency, FSI and Translator Directory**
+## Setting up the application(s) locally
+
+### Create folder structure for local storage and change owner accordingly
+
+    sudo mkdir -p /usr/local/var/onegov/depot
+    sudo mkdir -p /usr/local/var/onegov/files
+    sudo chown -R $USER:$USER /usr/local/var/onegov
+
+### Configure token
+
+add the mapbox token to you onegov.yml (generic)
+
+    mapbox_token: '<token>'
+
+> NOTE: Make sure you don't miss the single quotes
+
+**1) Use existing organisation - Transfer remote database to local setup**
+
+This will transfer the databases from remote to your local setup e.g. Community Ottenbach and many more
```

Comment:
Please do not add information about custom deployment and customers such as Ottenbach, erebos.seantis.ch etc.

Reply:
done